wrapper/src/main.cpp

```diff
@@ -159,6 +161,30 @@ int run_wrapper(char *fmt, int width, int height, char *ref_path, char *dis_path
         goto fail_or_end;
     }
 
+    if (fseek(s->ref_rfile, 0, SEEK_END))
+    {
+        fprintf(stderr, "fseek failed.\n");
+        ret = 1;
+        goto fail_or_end;
+    }
+
+    if ((file_size = ftell(s->ref_rfile)) == -1)
+    {
+        fprintf(stderr, "ftell failed.\n");
+        ret = 1;
+        goto fail_or_end;
+    }
+
+    rewind(s->ref_rfile);
+
+    frame_size = width * height + s->offset;
+    if (!strcmp(fmt, "yuv420p10le") || !strcmp(fmt, "yuv422p10le") || !strcmp(fmt, "yuv444p10le"))
+    {
+        frame_size *= 2;
+    }
+
+    s->num_frames = file_size / frame_size;
```

Comment:
I see the way you determine the num_frames is to have a first pass of the file size, and then use the file size divided by the frame size. This works when the input is YUV files, but probably won't work if it operates in the stream mode (for example, having libvmaf as a filter for FFmpeg), where the input is fed from an instantaneously decoded stream  of compressed video. 

My recommendation is not to to print out the num_frames as the denominator.

Reply:
From what I see, the `run_wrapper` function in `wrapper/src/main.cpp ` is exclusive invoked by vmafossexec. FFmpeg invokes `compute_vmaf` in `wrapper/src/libvmaf.cpp`, which doesn't go through this `run_wrapper` function. Nevertheless, I could skip the file size determination and set `num_frames` to -1 when "-" (pipe) is used as input to indicate that the total frames is unknown. Is that acceptable to you?

Reply:
Got it. That works. Thanks!